dynamicprefixvici.c

```diff
@@ -21,12 +30,44 @@ static char* getEnvironmentValue(const char* value);
 */
 static vici_req_t* createMessage(char* poolName, char* addressPool);
 
-int main(void)
+/**
+ * @brief get all command line arguments an passes gives it back in a struct.
+ * If unsupportet arguments are given, it terminates the program
+ * 
+ * @param argc 
+ * @param argv 
+ * @param arguments 
+ */
+static void ParseCommandLineArguments(int argc, char** argv, CommandLineArguments* arguments);
+
+int main(int argc, char** argv)
 {
-    char* newPrefix = getEnvironmentValue("new_dhcp6_ia_pd1_prefix1");
-    char* oldPrefix = getEnvironmentValue("old_dhcp6_ia_pd1_prefix1");
+    // Parse command line arguments
+    CommandLineArguments arguments;
+    ParseCommandLineArguments(argc, argv, &arguments);
+
+    // Set default values
+    char* newPrefixEnv = "new_dhcp6_ia_pd1_prefix1";
+    char* oldPrefixEnv = "old_dhcp6_ia_pd1_prefix1";
     char* poolName = "global-ipv6";
```

Comment:
Can we make is such that the poolName is a required command line argument, and not have a default value for the poolName?